docs/conf.py

```diff
@@ -29,10 +29,15 @@
 import datetime
 import sys
 
-ON_RTD = os.environ.get('READTHEDOCS') == 'True'
+try:
+    from sphinx_astropy.conf.v1 import *
+except ImportError:
+    print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+    sys.exit(1)
+
 ON_TRAVIS = os.environ.get('TRAVIS') == 'true'
 
-if ON_RTD:
+if on_rtd:
```

Comment:
Should be `ON_RTD`

Reply:
No, we now import this from sphinx_astropy and it's lower case.

Reply:
OH FAIR ENOUGH I SHOULD HAVE CHECKED